src/constants/selectionColumn.tsx

```diff
@@ -22,6 +22,9 @@ export const selectionColumn: ColumnDef<unknown> = {
             onChange={row.getToggleSelectedHandler()}
         />
     ),
+    meta: {
+        hideInSettings: true,
```

Comment:
Add the same meta in `actionsColumn` and `dragHandleColumn`

Reply:
Added